package.json

```diff
@@ -76,11 +76,11 @@
     "lerna": "4.0.0",
     "lint-staged": "11.0.0",
     "prettier": "2.2.1",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
Will [this](https://reactjs.org/blog/2020/10/20/react-v17.html#changes-to-event-delegation) impact anything?

Reply:
As I know @vidvidvid tried and it didnt create any problem.

Reply:
syntax wise it shouldn't create any problem or change.

Reply:
so if everything else looks good, i'd say we're ready to merge.